cmd/aws-iam-authenticator/root.go

```diff
@@ -135,6 +136,20 @@ func getConfig() (config.Config, error) {
 		return cfg, errors.New("Invalid partition")
 	}
 
+	//add validation for DynamicFile BackendMode and DynamicFilePath, they should appear together
```

Comment:
Style nit: space after // in comments.

Wording nit: slightly more succint description.
```
// DynamicFile BackendMode and DynamicFilePath are mutually inclusive.
```